apps/platform/src/sections/target/ChemicalProbes/Body.jsx

```diff
@@ -180,19 +105,16 @@ function Body({ definition, id, label: symbol }) {
       request={request}
       renderDescription={() => <Description symbol={symbol} />}
       renderBody={data => {
-        // sort probes manually as we need a double custom sort based on quality and origin
-        const sortedProbes = _.sortBy(data.target.chemicalProbes, [
```

Comment:
Hi @LucaFumis, sorry I didn't read your update before it was merged.
Ideally, we still want to sort on these parameters - especially in the case of probes of high quality. I meant to add the score sorting on top of the other two. 
I don't think it is a reason to halt the release process, however.